src/app/login/page.tsx

```
@@ -0,0 +1,127 @@
+            <div className="relative mx-auto flex w-[100%] items-center">
+              <div className="flex-1 border-2 border-lime-500"></div>
+              <span className="mx-4 font-semibold">or</span>
+              <div className="flex-1 border-2 border-lime-500"></div>
+            </div>
+          </div>
+        </div>
+
+        {/* Login Form */}
+        <form className="space-y-4">
+          <div className="space-y-2">
+            <label className="text-sm text-gray-600">Email Address</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)